### DISCUSSION TITLE
Notification

### DISCUSSION THREAD

**Discussion by newcc845** [dotnet/maui#2380] [Q&A]
Would like to ask, does MAUI have notification features? So far I did use xamarin way to code but so far not successful, can anyone can pin point the direction?

Thank you in advance.

**Comment by vhugogarcia** [comment]
You could use firebase cloud messaging for push notifications. However, I am not sure if firebase library is going to be available for .NET MAUI at launch 🚀. 

Anybody knows any news about GoogleApis for .NET MAUI?

**Reply by somoreingold** [reply]
Definitely interested in this as well; not just notifications, but also firebase analytics etc.

**Reply by KarlSearl** [reply]
this post seems like the user had success 

https://github.com/dotnet/maui/discussions/6557


I would have a look at that and perhaps contact that user for more help.

Let me know if you find a decent source of information as I will also need this feature.

**Reply by jzhao8402** [reply]
Really want to know if any user had success on FCM push notification.
